src/pages/landing/ebooks/components/_header.js

```diff
@@ -71,7 +71,7 @@ const HeaderImage = styled(QueryImage)`
 
     @media ${device.tabletS} {
         width: ${(props) => (props.imgWidth < 600 ? '115%' : '105%')};
-        height: ${(props) => (props.imgWidth < 600 ? '105%' : '105%')};
+        height: ${(props) => props.imgWidth < 600 && '105%'};
```

Comment:
If the `imgWidth` is <600, then it's gonna set the height to false. 
I think you can just set the height to `105%` without checking the condition.